docs/setup/kubeedge_configure.md

```diff
@@ -0,0 +1,369 @@
+        mode: 0 # 0: internal mqtt broker enable only. 1: internal and external mqtt broker enable. 2: external mqtt broker
+    enable only.
+        qos: 0 # 0: QOSAtMostOnce, 1: QOSAtLeastOnce, 2: QOSExactlyOnce.
+        retain: false # if the flag set true, server will store the message and can be delivered to future subscribers.
+        session-queue-size: 100 # A size of how many sessions will be handled. default to 100.
+    
+    edgehub:
+        websocket:
+            url: wss://0.0.0.0:10000/e632aba927ea4ac2b575ec1603d56f10/edge-node/events 
+            certfile: /etc/kubeedge/certs/edge.crt
+            keyfile: /etc/kubeedge/certs/edge.key
+            handshake-timeout: 30 #second
+            write-deadline: 15 # second
+            read-deadline: 15 # second
+        quic:
+            url: 127.0.0.1:10001
+            cafile: /etc/kubeedge/ca/rootCA.crt
+            certfile: /etc/kubeedge/certs/edge.crt
+            keyfile: /etc/kubeedge/certs/edge.key
+            handshake-timeout: 30 #second
+            write-deadline: 15 # second
+            read-deadline: 15 # second
+        controller:
+            protocol: websocket # websocket, quic
+            heartbeat: 15  # second
+            project-id: e632aba927ea4ac2b575ec1603d56f10
+            node-id: edge-node
+    
+    edged:
+        register-node-namespace: default
+        hostname-override: edge-node
+        interface-name: eth0
+        edged-memory-capacity-bytes: 7852396000
+        node-status-update-frequency: 10 # second
+        device-plugin-enabled: false
+        gpu-plugin-enabled: false
+        image-gc-high-threshold: 80 # percent
+        image-gc-low-threshold: 40 # percent
+        maximum-dead-containers-per-container: 1
+        docker-address: unix:///var/run/docker.sock
+        runtime-type: docker
+        remote-runtime-endpoint: unix:///var/run/dockershim.sock
+        remote-image-endpoint: unix:///var/run/dockershim.sock
+        runtime-request-timeout: 2
+        podsandbox-image: kubeedge/pause:3.1 # kubeedge/pause:3.1 for x86 arch , kubeedge/pause-arm:3.1 for arm arch, kubeedge/pause-arm64 for arm64 arch
+        image-pull-progress-deadline: 60 # second
+        cgroup-driver: cgroupfs
+        node-ip: ""
+        cluster-dns: ""
+        cluster-domain: ""
+        
+        mesh:
+            loadbalance:
+                strategy-name: RoundRobin
+```
+
+#### Modification in edge.yaml
+
+1. Update the IP address of the master in the `edgehub.websocket.url` field. You need set cloudcore ip address.
+
+    ```yaml
+    url: wss://0.0.0.0:10000/e632aba927ea4ac2b575ec1603d56f10/edge-node/events
+    ```
+    to 
+
+    ```yaml
+    url: wss://<X.X.X.X>:10000/e632aba927ea4ac2b575ec1603d56f10/edge-node/events
+    ```
+    where X.X.X.X is your cloudcore IP Address.
+
+2. Update the IP address of the master in the `edgehub.quic.url` field. You need set cloudcore ip address.
```

Comment:
only if you want to use quic an then edgehub.controller.protocol has also be updated.